torch/_subclasses/meta_utils.py

```diff
@@ -313,7 +313,7 @@ def sym_sizes_strides_storage_offset(t, src):
                         shape_env,
                         callback,
                         source=AttrSource(source, "_base"),
-                        dynamic_dims=base_dynamic_dims,
+                        dynamic_dims=None,
```

Comment:
What's going on here?

Reply:
This looks like a hack to get around the mismatch between `len(base_dynamic_dims)` and dims of the values and offsets that causes problems for NT (also see https://github.com/pytorch/pytorch/pull/112185#discussion_r1373715054).

For the purposes of this PR, this should definitely be removed.

Reply:
Whoops, fixed!